datashare-app/src/main/java/org/icij/datashare/com/bus/amqp/Event.java

```diff
@@ -0,0 +1,42 @@
+package org.icij.datashare.com.bus.amqp;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.icij.datashare.json.JsonObjectMapper;
+import org.icij.datashare.time.DatashareTime;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Base event for AMQP.
+ * It provides creation timestamp and ttl that can be used for error handling and republishing/DLQ strategies.
+ */
+public class Event implements Serializable {
```

Comment:
I'm wondering about the `Event` naming, I was thinking about something more generic like `Message`, which could be describe as task to perform, a task related event or anything else.